packages/app/utils/object.ts

```diff
@@ -10,12 +10,31 @@ export function clearEmpties(o: any) {
             delete o[k] // clear undefined or null values
         }
 
+        if (Array.isArray(o[k])) {
```

Comment:
What would you think about using something like Lodash's `cloneDeep` or Immer's `produce` to prevent side effects, since this is a general object util?

Reply:
Sure, makes sense. Added cloneDeep in